InstabugSample/metro.config.js

```diff
@@ -5,7 +5,15 @@
  * @format
  */
 
+const path = require('path');
+const escape = require('escape-string-regexp');
+const exclusionList = require('metro-config/src/defaults/exclusionList');
+
+const root = path.resolve(__dirname, '..');
+const modules = ['react', 'react-native', '@babel/runtime'];
```

Comment:
IMO, extracting `react` and `react-native` from `peerDependencies` will decouple this file from the changes made to the SDK's `package.json`.

Reply:
Good call, but the thing is that we don't actually have any `peerDependencies` in the SDK's `package.json` so I added them as an array in the `metro.config.js` file, another solution would be to add them as `peerDependencies` and get them dynamically in `metro.config.js`.

Reply:
We can add them for now with unconstrained versions `"*"`.